spec/clipboard.spec.js

```diff
@@ -111,27 +137,122 @@ describe('clipboard.spec: clipboard namespace tests', () => {
             let resultJson = JSON.parse(result);
             assert.equal(resultJson.clipboardText1, resultJson.clipboardText2);
         });
+
         it('returns text with leading and trailing whitespaces intact', async () => {
-            let exitCode = runner.run(`
+            runner.run(`
                 await Neutralino.clipboard.writeText('  Test value  ');
                 let clipboardText = await Neutralino.clipboard.readText();
                 await __close(clipboardText);
             `);
             assert.equal(runner.getOutput(), '  Test value  ');
         });
+
         it('returns text with special characters intact', async () => {
-            let exitCode = runner.run(`
+            runner.run(`
                 await Neutralino.clipboard.writeText('@#$%^&*☁☀☊☄');
                 let clipboardText = await Neutralino.clipboard.readText();
                 await __close(clipboardText);
             `);
             assert.equal(runner.getOutput(), '@#$%^&*☁☀☊☄');
         });
+
+        it('returns text with newline characters intact', async () => {
+            runner.run(`
+                await Neutralino.clipboard.writeText('Line1\\nLine2\\nLine3');
+                let clipboardText = await Neutralino.clipboard.readText();
+                await __close(clipboardText);
+            `);
+            assert.equal(runner.getOutput(), 'Line1\nLine2\nLine3');
+        });
     });
 
+    describe('clipboard.writeImage', () => {
+        it('writes an image to the clipboard successfully', async () => {
+            let image = {
+                width: 400,
+                height: 400,
+                bpp: 32,
+                bpr: 400,
+                redMask: 0xff0000,
+                greenMask: 0x00ff00,
+                blueMask: 0x0000ff,
+                redShift: 16,
+                greenShift: 8,
+                blueShift: 0,
+                data: new ArrayBuffer(40000)
+            };
+            runner.run(`
+                await Neutralino.clipboard.writeImage(${JSON.stringify(image)});
```

Comment:
`ArrayBuffer` instance won't work with JSON.stringify properly, so please consider moving `let image...` to `runner.run()` from Node. Thanks